mesonbuild/build.py

```diff
@@ -1140,6 +1140,11 @@ def add_include_dirs(self, args):
             if not isinstance(a, IncludeDirs):
                 raise InvalidArguments('Include directory to be added is not an include directory object.')
             ids.append(a)
+        if set_is_system is None:
+            system_flags = 'preserve'
```

Comment:
__[Flake8]__ [F841] local variable 'system_flags' is assigned to but never used ([link](https://sider.review/gh/repos/19784232/pulls/5953#issue-4627265))

<sub>posted by Sider</sub>